src/app/not-found.tsx

```diff
@@ -1,5 +1,28 @@
+import Image from "next/image";
+import image1 from "../../public/404.webp";
+import Link from "next/link";
+
 const NotFound = () => {
-  return <div>404 Page Not Found</div>;
+  return (
+    <div className="relative h-screen w-full overflow-hidden bg-hearts-beige">
+      <Image
+        src={image1}
+        alt="404 Background"
+        fill
+        style={{ objectFit: "contain" }}
+        className="pointer-events-none z-10"
+      />
+
+      <div className="absolute inset-0 flex items-end justify-center">
+        <Link
```

Comment:
Make sure to add tailwindcss duration to the button. What duration do is it create a better transition when user hover on the button.